getRoster.py

```diff
@@ -0,0 +1,39 @@
+from showRoster import *
+import discord
+import os
+def findTeam(team):
+  final = "";
+  if(contains(team) == 1):
+    return "```"+team+" roster\n"+getRoster(team.split(" ")[1])+"```"
+  
+  elif(containsLast(team) == 1):
+    return "```"+team+" roster\n"+getRoster(team)+"```"
+  
+
+def contains(team):
```

Comment:
Instead of using a function to see if a team is in team names, another way to do this is to create a set with all the names in it and check to see if a name is in that set: 
```
def findTeam(team):
     shortTeamName = team.split()[-1] # get the short team name 
                                      # by getting the last word of the string
     owlTeams = set(['reign', 'uprising', 'fuel',...])
     if shortTeamName in owlTeams:
        return # return value here
 ```
This will create an O(1) search time (not that it really matters with this), as well as eliminate the need for the contains() and containsLast() functions.